src/EnergyPlus/ZoneEquipmentManager.cc

```diff
@@ -3993,14 +3994,13 @@ namespace EnergyPlus::ZoneEquipmentManager {
 
                 Real64 FinalTotalReturnMassFlow = 0;
                 CalcZoneReturnFlows(state, ZoneNum, StdTotalReturnMassFlow, FinalTotalReturnMassFlow);
-
                 MassConservation(ZoneNum).RetMassFlowRate = FinalTotalReturnMassFlow;
 
                 // Set zone infiltration flow rate
                 if (ZoneAirMassFlow.InfiltrationTreatment != NoInfiltrationFlow) {
                     if (MassConservation(ZoneNum).InfiltrationPtr > 0) {
                         if (MassConservation(ZoneNum).IsOnlySourceZone || (ZoneAirMassFlow.InfiltrationZoneType == AllZones)) {
-                            ZoneInfiltrationMassFlowRate = MassConservation(ZoneNum).MixingSourceMassFlowRate +
+                            ZoneInfiltrationMassFlowRate = MassConservation(ZoneNum).MixingSourceMassFlowRate - MassConservation(ZoneNum).MixingMassFlowRate +
```

Comment:
This is am missing term added. This would have been caught early if we had a test cases with a thermal zone that is used as a source and receiving zone.

Reply:
:+1: